cleo_plugins/Audio/C3DAudioStream.cpp

```diff
@@ -6,6 +6,8 @@ using namespace CLEO;
 
 C3DAudioStream::C3DAudioStream(const char* filepath) : CAudioStream()
 {
+    static_assert(offsetof(C3DAudioStream, streamInternal) == 4, "C3DAudioStream compatibility with CLEO4 broken!");
```

Comment:
```suggestion
    // see https://github.com/cleolibrary/CLEO5/pull/230
    static_assert(offsetof(C3DAudioStream, streamInternal) == 4 && alignof(C3DAudioStream) == 4, "C3DAudioStream compatibility with CLEO4 broken!");
```

Reply:
Does `alignof(C3DAudioStream)` has any meaning in that case? All we care of is final offset of `streamInternal` in relation to instance pointer.

Reply:
Yes it has meaning. If you see above discussion, even though `streamInternal` had offset +4 in IDE the final offset was either +4 or +8 depending on the alignment of the class. This is why we got rid of 8-byte long fields.

Reply:
if you add `double` back, the `alignof` becomes 8 and assert fail, which is what we want.

Reply:
'offsetof' fails too, but during compilation time.